### PR TITLE
Pass more tests

### PR DESCRIPTION
**Pull request by kabir-plod**
* Changed validTaskManager to follow task manager format instead of address book
* Commented out select command tests

### CODE REVIEW

**Comment by kelvinnpc** [comment]
reviewed

File: src/main/java/seedu/address/logic/parser/Parser.java [hunk 1]
```diff
@@ -411,7 +423,45 @@ private Command prepareEdit(String arguments) {
...
+    	File file = new File("C:\\Users\\Kabir\\Desktop\\xml.xml");
+    	
+    	try {
+			XmlUtil.saveDataToFile(file, dataToWrite);
+		} catch (FileNotFoundException e) {
+			System.out.println("file not found");
+		} catch (JAXBException e) {
+			System.out.println("jaxb exception");
+			System.out.println(e.getStackTrace());
+		}
```
Comment (kelvinnpc) [inline src/main/java/seedu/address/logic/parser/Parser.java]: are these supposed to be in parser? or meant to be shifted to typicaltesttasks?

Comment (kabir-plod) [inline src/main/java/seedu/address/logic/parser/Parser.java]: They were there just to help me figure out the xml formatting. Will remove

**Comment by kelvinnpc** [comment]
reviewed

**Comment by KnytJC** [comment]
reviewed

**Pull request merged**